lib/cs_guide_web/templates/venue/show.html.eex

```diff
@@ -151,3 +61,30 @@
     </div>
   </section>
 </section>
+
+<!-- Google Maps -->
+<script>
+  function initMap() {
+    var london = {lat: 51.509865, lng: -0.118092};
+    var address = <%= raw Poison.encode!(@venue.postcode) %>
+    var map = new google.maps.Map(document.getElementById('map'), {zoom: 16, center: london});
+    var geocoder = new google.maps.Geocoder();
+    codeAddress(address, geocoder, map);
+  }
+
+  function codeAddress(address, geocoder, map) {
+    geocoder.geocode({'address': address}, function(results, status) {
+      if (status === 'OK') {
+        map.setCenter(results[0].geometry.location);
+        var marker = new google.maps.Marker({
+          map: map,
+          position: results[0].geometry.location
+        });
+      } else {
+        alert('Geocode was not successful for the following reason: ' + status);
```

Comment:
ideally, this error `alert` should also send an analytics event so we (devs/PO) know about the fail. 🤔